build.psake.ps1

```diff
@@ -70,8 +70,8 @@ Properties {
 
     # The root directory of the module source and tests.  It could be the workspace
     # root or a subdir such as src, module, <my-module-name>.
-    $SourceRootDir = "$PSScriptRoot/src"
-    $TestRootDir   = "$PSScriptRoot/test"
+    $SourceRootDir = Join-Path -Path $PSScriptRoot -ChildPath "src"
+    $TestRootDir   = Join-Path -Path $PSScriptRoot -ChildPath "test"
 
```

Comment:
That is pretty canonical throughout this code base and I don't see a need to change it.  The value of `$PSScriptRoot` is known to never end if a backslash.  I since PowerShell handles either \ or / the interpolated string approach has no issues I know and is more readable.